src/drivers/lis3mdl/lis3mdl.cpp

```diff
@@ -1178,38 +1178,38 @@ int LIS3MDL::calibrate(struct file *filp, unsigned enable)
 
 int LIS3MDL::check_scale()
```

Comment:
There's potentially a bit of ambiguity mixing int and bool return values, but it's not wrong.